### ISSUE TITLE
Server side chat failing

### ISSUE THREAD

**Issue by wallenfox** [magefree/mage#814]
Upgraded server to 1.3.0.dev2015-03-14v3 and get the following errors on the server.  Users can connect and setup and join games but can not play and there is no chat.  Every time a user tries to send a chat message another error like below is generated.

![capture](https://cloud.githubusercontent.com/assets/11548657/6723449/37235a74-cda5-11e4-9d35-32b45e871bcb.PNG)

**Comment by seedds** [comment]
same problem here since 0314v1.

Server is on Ubuntu 14 with openjdk7.

**Comment by wallenfox** [comment]
Ubuntu 14 with openjdk7   -  Same here.

**Comment by LevelX2** [comment]
It's true I switched to oracle 7 and since 3 days to oracle 8 on the server.
Also I created the jar with java 8 what also seem to matter. Oracle java has the advantage of java missin control.
Any reason on your side not to update to oracle java 8?

Nevertheless I will check if I can create the next update with java 7 again.


So a workaround would be to install and use java 8 on your linux servers.


https://www.digitalocean.com/community/tutorials/how-to-install-java-on-ubuntu-with-apt-get

**Add oracle java to apt repository:**
sudo apt-get install software-properties-common
sudo add-apt-repository ppa:webupd8team/java
sudo apt-get update

**Install oracle java 7 and/or 8:**
sudo apt-get install oracle-java7-installer
sudo apt-get install oracle-java8-installer

**Change java default version:**
sudo update-alternatives --config java
